updater/lib/dependabot/job.rb

```diff
@@ -270,8 +336,9 @@ def security_advisories_for(dependency)
       end
     end
 
+    sig { params(dependency: Dependabot::Dependency).returns(T::Array[String]) }
     def ignore_conditions_for(dependency)
-      update_config.ignored_versions_for(
+      @update_config.ignored_versions_for(
```

Comment:
```suggestion
      update_config.ignored_versions_for(
```

Reply:
Ah, so `def update_config` was actually private so there's no `attr_reader` for it which is why I used the `@` syntax. We could expose it as an `attr_reader` but it's not needed currently. Should we do that?

Reply:
You can make it a private `attr_reader` by defining after declaring `private`!

Reply:
:woah: